backend/src/main/kotlin/cz/loono/backend/schedule/TaskScheduler.kt

```diff
@@ -0,0 +1,16 @@
+package cz.loono.backend.schedule
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component("taskScheduler")
+class TaskScheduler @Autowired constructor(
+    private val examinationCancellationTask: ExaminationCancellationTask
+) {
+
+    @Scheduled(cron = "0 0 3 * * ?") // each day at 3AM
```

Comment:
Maybe extract cron expression this to the application.yaml and then inject it here?

Reply:
I already tried to use a definition in the yml but it didn't work. If I define a property and insert it to the annotation then it cannot be compiled.

Reply:
What about this?
In application.yaml:
`application.cancellation.scheduler.cron: "0 0 3 * * ?"`

And then in code itself:
`@Scheduled(cron = "\${application.cancellation.scheduler.cron}")`

Reply:
What about this?
In application.yaml:
`application.cancellation.scheduler.cron: "0 0 3 * * ?"`

And then in code itself:
`@Scheduled(cron = "\${application.cancellation.scheduler.cron}")`

Reply:
Ha, missing backslash :-) Thanks.